src/corg/developer-environment-setup/intellij-idea-setup.md

```diff
@@ -0,0 +1,85 @@
+---
+title: IntelliJ IDEA Setup
+labels:
+    - intellij
+    - idea
+---
+
+## Download and Install IntelliJ IDEA
+
+You can [download][idea-download] the latest version of IntelliJ IDEA from Jet
+Brains, the official website. It supports all the platforms including Mac OS,
+Windows, and Linux. IntelliJ has two editions: Community Edition and Ultimate
+Edition. Nuxeo sources code work with both of them. Once downloaded, install
+IntelliJ IDEA with your own preferences.
+
+## Import Nuxeo Source Code into IntelliJ IDEA
+
+{{#> callout type='note' }}
+
+We assume that you've cloned the Nuxeo source code successfully. If it's not the
+case, please go back to the previous section to see how to do it.
+
+{{/callout}}
+
+Before importing Nuxeo source code, you need to configure the VM options for
+importer to increase the importation capacity. Open IntelliJ, a welcome menu
+will be shown. On the right bottom of menu, click _Configure_ > _Preferences_.
+Then search `VM options for importer` and set it to:
+
+    -Xms1g -Xmx4g
+
+Now import Nuxeo source code as Maven project. In the default welcome menu,
+choose _Import Project_, then find the Nuxeo root folder and select the POM file
+in `$NX_HOME/pom.xml`. Afterwards, enable Maven import options as the following:
```

Comment:
`NX_HOME` is undefined and not clear (and not to be confused with `NUXEO_HOME` used in the server): it's the root of Nuxeo source code. Write `nuxeo/pom.xml` or `NUXEO_SOURCES` or...